### ISSUE TITLE
Wind datas wrongs when MPS

### ISSUE THREAD

**Issue by umpire274** [mivek/python-metar-taf-parser#4]
Hello.

I've parsed this METAR string

`UNAA 240830Z 34002MPS CAVOK M14/M18 Q1019 R02/190054 NOSIG RMK QFE741`

and I find two groups of datas into `Wind` object:

`_speed : 2`
`_direction : NNW`
`_degrees : 340`
`_gust : None`
`_min_variation : None`
`_max_variation : None`
`_unit : MPS`

`_speed : 5`
`_direction : S`
`_degrees : 190`
`_gust : None`
`_min_variation : None`
`_max_variation : None`
`_unit : KT`

This problem is generated by regex string into class `WindCommand`:

`regex = r'(VRB|\d{3})(\d{2})G?(\d{2})?(KT|MPS|KM\/H)?'`

that catches two strings matching the regex: `34002MPS` and `19005`

I've solved deleting the last `?` in regex string.

`regex = r'(VRB|\d{3})(\d{2})G?(\d{2})?(KT|MPS|KM\/H)'`

Check if this is true also for your test.

**Comment by mivek** [comment]
Hello @umpire274 

There is indeed a problem with the parsing. 
The real problem here is that the token `R02/190054` should be a runwayInfo, the regex should have recognized the first part `R02/`.
The part `190054` indicates a deposit on the runway, the parser does not handle the parsing of deposit for now. I will be working on that soon.

I am not sure i can simply remove the last `?` of the WindCommand regex because metar do not necessarily contain the unit.

**Comment by umpire274** [comment]
Hi @mivek .

This is another problem that I had noticed and was trying to solve it by modifying the 'RunwayCommand' class by inserting a special regex for the string 'R02/123465' but at the moment unsuccessfully. 

The bug I've mentioned in the issue is relative to string '34002MPS' for the wind, beacause in ex URSS they use m/s ('MPS') instead KT or KM/H.

The regex for the Wind string is in metar_taf_parser/command/common.py and is `regex = r'(VRB|\d{3})(\d{2})G?(\d{2})?(KT|MPS|KM\/H)?'`

I had solved the problem **deleting the '?' at the end of regex string**.

If you'll try it you'll see the solution.

For the problem about `R02/190054` we need a specific issue.

I hope I was clear in my explanation.

Alex.

> Hello @umpire274
> 
> There is indeed a problem with the parsing.
> The real problem here is that the token `R02/190054` should be a runwayInfo, the regex should have recognized the first part `R02/`.
> The part `190054` indicates a deposit on the runway, the parser does not handle the parsing of deposit for now. I will be working on that soon.
> 
> I am not sure i can simply remove the last `?` of the WindCommand regex because metar do not necessarily contain the unit.
